apps/openssl-vms.cnf

```diff
@@ -1,7 +1,9 @@
 #
 # OpenSSL example configuration file.
-# This is mostly being used for generation of certificate requests.
+# See doc/man5/config.pod for more info.
```

Comment:
Oh!  You should probably refer to `openssl-fipsinstall(1)` in the SEE ALSO in `doc/man5/config.pod`

Reply:
I added that in this PR..